expr/agg/collect.go

```diff
@@ -92,29 +93,30 @@ func (c *Collect) buildUnion(zctx *zed.Context, selectors map[zed.Type]int) (zed
 	}
 	unionType := zctx.LookupTypeUnion(types)
 	arrayType := zctx.LookupTypeArray(unionType)
-	return zed.Value{arrayType, b.Bytes()}, nil
+	return zed.NewValue(arrayType, b.Bytes())
 }
 
-func (c *Collect) ConsumeAsPartial(zv zed.Value) error {
+func (c *Collect) ConsumeAsPartial(val *zed.Value) {
```

Comment:
Did we decide to go with val of zv?

Reply:
@mattnibs: @mccanne and I think we should, both because it's less cryptic and for agreement with `vals := batch.Values()`.